Makefile

```diff
@@ -47,3 +47,8 @@ tc-worker-env:
 tc-worker:
 	CGO_ENABLED=$(CGO_ENABLE) GOARCH=amd64 go build
 	docker build -t taskcluster/tc-worker -f tc-worker.Dockerfile .
+
+lint:
+	go get github.com/alecthomas/gometalinter
+	gometalinter --install
+	$(GOPATH)/bin/gometalinter --deadline 10m $$(go list ./... | grep -v /vendor/ | sed 's/github.com\/taskcluster\/taskcluster-worker/./')
```

Comment:
Lets not run everything... here is my settings from atom:
`--vendor, --disable-all, --enable=vet, --enable=misspell, --enable=vetshadow, --enable=golint, --enable=ineffassign, --enable=goconst, --tests`

I'm pretty sure we've debated these at some point... I'm open to adding more, but let's add one at the time...